src/chart/sankey/SankeyView.ts

```diff
@@ -283,7 +285,9 @@ class SankeyView extends ChartView {
             const focus = emphasisModel.get('focus');
             toggleHoverEmphasis(
                 rect,
-                focus === 'adjacency' ? node.getAdjacentDataIndices() : focus,
+                focus === 'adjacency' ? node.getAdjacentDataIndices()
+                : focus === 'trajectory' ? node.getFullPathDataIndices()
+                : focus,
```

Comment:
Code style preference

```suggestion
                focus === 'adjacency'
                    ? node.getAdjacentDataIndices()
                    : focus === 'trajectory'
                        ? node.getFullPathDataIndices()
                        : focus,
```